### PR TITLE
Wip improvements

### PR DESCRIPTION
**Pull request by jorgelsalas**
This PR implements a fragment with PDF417 scanning capabilities.

### CODE REVIEW

File: materialbarcodescanner/src/main/java/com/edwardvanraak/materialbarcodescanner/BarcodeGraphic.java [hunk 1]
```diff
@@ -97,6 +96,7 @@ public void draw(Canvas canvas) {
         /**
          * Draw the top left corner
          */
+        int mCornerWidth = 64;
```
Comment (jdgarita) [inline materialbarcodescanner/src/main/java/com/edwardvanraak/materialbarcodescanner/BarcodeGraphic.java:99]: is this a default value?

File: materialbarcodescanner/src/main/java/com/edwardvanraak/materialbarcodescanner/MaterialBarcodeScanner.java [hunk 2]
```diff
@@ -64,42 +67,62 @@ public void onBarcodeScannerResult(Barcode barcode) {
...
+
+            // new code - attempting to start a fragment
+            FragmentManager fragmentManager = ((AppCompatActivity) materialBarcodeScannerBuilder.getActivity()).getSupportFragmentManager();
+            materialBarcodeScannerFragment = MaterialBarcodeScannerFragment.newInstance(null);
+
+            int id = materialBarcodeScannerBuilder.getRootView().getId();
+
+            fragmentManager.beginTransaction()
+                .replace(id, materialBarcodeScannerFragment)
+                .addToBackStack(MaterialBarcodeScannerFragment.class.getSimpleName())
```
Comment (jdgarita) [inline materialbarcodescanner/src/main/java/com/edwardvanraak/materialbarcodescanner/MaterialBarcodeScanner.java:89]: Do we really need to call an addToBackStack?

**Review by thitu — APPROVED** [review]

File: materialbarcodescanner/src/main/java/com/edwardvanraak/materialbarcodescanner/BarcodeGraphic.java (hunk 1 above)
Comment (jorgelsalas) [reply materialbarcodescanner/src/main/java/com/edwardvanraak/materialbarcodescanner/BarcodeGraphic.java:99]: Seems to be. Looks like this class is supposed to draw a barcode when it's scanned, but I don't think this code is executed when scanning the pdf417...

File: materialbarcodescanner/src/main/java/com/edwardvanraak/materialbarcodescanner/MaterialBarcodeScanner.java (hunk 2 above)
Comment (jorgelsalas) [reply materialbarcodescanner/src/main/java/com/edwardvanraak/materialbarcodescanner/MaterialBarcodeScanner.java:89]: There's actually no need to, but this method is overridden by a class in the tru-dl sdk in which we don't add the fragment to the back stack.

Comment (jdgarita) [reply materialbarcodescanner/src/main/java/com/edwardvanraak/materialbarcodescanner/MaterialBarcodeScanner.java:89]: yeah I saw the another call, cool!

**Review by jdgarita — APPROVED** [review]

**Pull request merged**